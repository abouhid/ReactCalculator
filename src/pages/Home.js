import React from 'react';

const Home = () => (

  <div className="quote-container disp">
    <h2>Welcome to my Calculator App! </h2>
    <h3>Get yourself cozy, bring a hot cup of tea and enjoy calculating whatever you want.</h3>
    <h4>
      Planning the expanses of your next trip? We got your back!
      <br />
      <br />

      You want to check if little Timmy is spending too much on in-game skins?
      We are here for ya!

      <br />
      <br />
      You bet on the wrong horse and needs to move to Bolivia? You&apos;re on your own!
    </h4>
    <h3>We hope you enjoy your stay!</h3>
  </div>
);

export default Home;
