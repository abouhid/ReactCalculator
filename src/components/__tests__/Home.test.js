import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../pages/Home';

describe('Home', () => {
  it('Tests the rendering of components', () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});