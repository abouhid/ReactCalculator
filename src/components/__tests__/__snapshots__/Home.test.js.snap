// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Tests the rendering of components 1`] = `
<div
  className="quote-container disp"
>
  <h2>
    Welcome to my Calculator App! 
  </h2>
  <h3>
    Get yourself cozy, bring a hot cup of tea and enjoy calculating whatever you want.
  </h3>
  <h4>
    Planning the expanses of your next trip? We got your back!
    <br />
    <br />
    You want to check if little Timmy is spending too much on in-game skins? We are here for ya!
    <br />
    <br />
    You bet on the wrong horse and needs to move to Bolivia? You're on your own!
  </h4>
  <h3>
    We hope you enjoy your stay!
  </h3>
</div>
`;

exports[`Home component renders correctly 1`] = `
<div
  className="quote-container disp"
>
  <h2>
    Welcome to my Calculator App! 
  </h2>
  <h3>
    Get yourself cozy, bring a hot cup of tea and enjoy calculating whatever you want.
  </h3>
  <h4>
    Planning the expanses of your next trip? We got your back!
    <br />
    <br />
    You want to check if little Timmy is spending too much on in-game skins? We are here for ya!
    <br />
    <br />
    You bet on the wrong horse and needs to move to Bolivia? You're on your own!
  </h4>
  <h3>
    We hope you enjoy your stay!
  </h3>
</div>
`;
